### ISSUE TITLE
Adjust_sdk lib has been deprecated after upgrading Flutter upto ver.2.5.2 

### ISSUE THREAD

**Issue by cungtrang0978** [adjust/flutter_sdk#64]
Console:
```console
C:\Flutter\flutter\.pub-cache\hosted\pub.dartlang.org\adjust_sdk-4.29.1\android\src\main\java\com\adjust\sdk\flutter\AdjustSdk.java:50: warning: [deprecation] Registrar in PluginRegistry has been deprecated
import io.flutter.plugin.common.PluginRegistry.Registrar;
                                              ^
C:\Flutter\flutter\.pub-cache\hosted\pub.dartlang.org\adjust_sdk-4.29.1\android\src\main\java\com\adjust\sdk\flutter\AdjustSdk.java:121: warning: [deprecation] Registrar in PluginRegistry has been deprecated
    public static void registerWith(Registrar registrar) {
```
Hope this will be fixed soon

Thank you

**Comment by mericgerceker** [comment]
I have the same error :(

**Comment by uerceg** [comment]
Hey guys,

Thanks for reporting this. Indeed, these warnings are being flagged during build process. From what I can see, they are not errors, rather just warnings which still allow app to be built and run. But nevertheless, we'll take a look into addressing these.

**Comment by uerceg** [comment]
Hi @cungtrang0978,

We have just released [Flutter SDK v4.29.2](https://github.com/adjust/flutter_sdk/releases/tag/v4.29.2) which should hopefully address this issue. Feel free to give it a shot and in case issue still persists, feel free to comment / reopen.

Cheers.
